js/app/moltresEngine.js

```diff
@@ -196,6 +205,16 @@ placerat varius non id dui.',
         return Eknc.SetObjectModel.new_from_props(data);
     },
 
+    _generate_audio_object: function (data) {
+        data.synopsis = this._SYNOPSIS;
+        data.content_type = 'audio/webm';
+        data.title = AUDIO_TITLE;
+        data.ekn_id = IMAGES_DIR + VIDEO_FILENAME + new Array(this._audio_count++).fill('/').join('');
```

Comment:
Yes, I was just following the same scheme we use for videos, but I can fix it there as well.